src/main/java/com/salesforce/mirus/config/SourceConfigDefinition.java

```diff
@@ -96,6 +96,12 @@
       "org.apache.kafka.connect.converters.ByteArrayConverter",
       ConfigDef.Importance.MEDIUM,
       "Converter class to apply to source record headers"),
+  COMMIT_FAILURE_RESTART_MS(
```

Comment:
This is related to `OFFSET_COMMIT_INTERVAL_MS_CONFIG` and should be greater than it, always.

Reply:
Yeah, true. The problem is `OFFSET_COMMIT_INTERVAL_MS_CONFIG` belong to the [WorkerConfig](https://github.com/apache/kafka/blob/2.6.1/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java#L136), here we are working at the `Task/SourceTask` level, it's not easy to get the upper level config info unless it is passed in which is not.

Reply:
In that case a comment should suffice.